lib/phases/phase.js

```diff
@@ -505,15 +505,17 @@ Phase.prototype.calculateStats = function() {
         .forEach(function(key) {
           var values = contextResults[key];
           var mean = stats.mean(values);
+          var stdev = stats.stdev(values);
 
           statistics[contextKey].push({
             Metric: key,
             Mean: mean,
             Median: stats.median(values),
             Min: Math.min.apply(Math, values),
+            '95th Perc.': stats.percentile(values, 0.95) || mean,
```

Comment:
The tough thing is the current dependency with how things are done right now, with the way things should actually be. For example, right now all of our metrics in the dashboards display p95. All of our baselines for v2.2 are calculated using p95. All our regression detection scripts query for p95.

All that is to say: we could easily switch everything over to use mean, but we would have to change the messaging around our releases to say that we are abandoning p95 for another alternative. Thoughts?

FYI, here is all the stats for v2.2 I gathered. https://gist.github.com/eliperelman/5aac3b94e2f815dbb827